### PR TITLE
Added colors and countdown timer

### PR DESCRIPTION
**Pull request by sepiatonal**

### CODE REVIEW

File: src/main/com/github/soerxpso/portalfix/PortalFix.java [hunk 1]
```diff
@@ -150,7 +154,7 @@ public void run() {
 						public void run() {
 							Player qp = Bukkit.getPlayer(this.ap);
 							if (qp == null) return; // else, they logged off.
-							qp.sendMessage(ChatColor.RED + Config.getPostMessage());
+							qp.sendMessage(doColors(Config.getPostMessage()));
```
Comment (ProgrammerDan) [inline src/main/com/github/soerxpso/portalfix/PortalFix.java]: So I found a function over the weekend:

https://hub.spigotmc.org/javadocs/bukkit/org/bukkit/ChatColor.html#translateAlternateColorCodes(char,%20java.lang.String)

Sigh. We should probably use that :3 -- I'll be "updating" HiddenOre with it later on as well.

File: src/main/com/github/soerxpso/portalfix/PortalFix.java [hunk 2]
```diff
@@ -98,11 +97,15 @@ public void run() {
...
 		private Location l;
+		private Countdown countdown;
 		
 		public TeleportIfStill(UUID p, Location l) {
 			this.p = p;
 			this.l = l;
 			getServer().getPluginManager().registerEvents(this, PortalFix.getPlugin());
+			countdown = new Countdown((Config.getWaitTime() - Config.getCountdownInterval()) * 20, 
+					Config.getCountdownInterval() * 20, getPlugin().getServer().getPlayer(p));
+			countdown.runTaskLater(PortalFix.getPlugin(), Config.getCountdownInterval() * 20);
```
Comment (ProgrammerDan) [inline src/main/com/github/soerxpso/portalfix/PortalFix.java]: IIRC you should be able to use runTaskTimer() here; just keep your count of runs, and have it cancel itself when you've notified the user enough times.

File: src/main/com/github/soerxpso/portalfix/PortalFix.java [hunk 3]
```diff
@@ -160,6 +164,40 @@ public void run() {
...
+		
+		//Devoted is not responsible for the possible results of taking life advice from method names
+		public void killSelfAndChildren() {
+			if(child == null) {
+				this.cancel();
+			}else {
+				child.killSelfAndChildren();
+			}
+		}
+	}
```
Comment (ProgrammerDan) [inline src/main/com/github/soerxpso/portalfix/PortalFix.java]: Using the runTaskTimer should let you avoid this method. Although it is clever :P

Comment (sepiatonal) [inline src/main/com/github/soerxpso/portalfix/PortalFix.java]: Alright, I'll fix it tonight

Comment (ProgrammerDan) [inline src/main/com/github/soerxpso/portalfix/PortalFix.java]: Thanks!

**Comment by sepiatonal** [comment]
Done

**Pull request merged**